tests/unit/lms/views/basic_lti_launch_test.py

```diff
@@ -279,11 +267,7 @@ def test_it_configures_frontend_grading(
         via_url,
         ModuleItemConfiguration,
     ):
-        pyramid_request.params = {
-            "resource_link_id": "TEST_RESOURCE_LINK_ID",
-            "tool_consumer_instance_guid": "TEST_TOOL_CONSUMER_INSTANCE_GUID",
-            **lti_outcome_params,
```

Comment:
In this case the view being tested doesn't even need `"resource_link_id"` or `"tool_consumer_instance_guid"`. The test was adding them for no apparent reason

Reply:
Actually it doesn't need the `lti_outcome_params` either